.env.example

```diff
@@ -7,6 +7,10 @@ REACT_APP_HOME_NATIVE_NAME=POA
 REACT_APP_HOME_NETWORK_NAME="POA Sokol"
 REACT_APP_FOREIGN_NETWORK_NAME=Kovan
 
+# Uncomment the constant for the network that don't support events
+#REACT_APP_HOME_WITHOUT_EVENTS=1
```

Comment:
Does it make sense to use "true' instead of "1"?

Reply:
Done

Reply:
@patitonar, @akolotov changed. Now by default both constants are `false`, and the user will need to changed them to `true` to inform of a network _without events_